specification/e-referrals-service-api.yaml

```diff
@@ -1288,7 +1292,7 @@ paths:
         
         You need to retrieve available appointment slots for a service using [[HYPERLINK_A015]].
 
-        You need to use the 'Unique Booking Reference Number' (UBRN) of a referral request in order to use this endpoint.
+        You need to use the `Unique Booking Reference Number` (UBRN) of a referral request in order to use this endpoint.
```

Comment:
Should this be displayed as a code block? I would have thought single quotes would have made more sense here or no quotes at all.

Reply:
This was https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-60881 . It's just bringing this one in line with the formatting Unique Booking Reference Number has in other endpoint's description.